src/backend/internal/controller/user/profile_picture.go

```diff
@@ -88,6 +88,18 @@ func GetProfilePictureUrl(profilePictureKey *string) string {
 }
 
 // UploadProfilePicture handles the HTTP request for uploading a profile picture.
+//
+// @Summary      Upload profile picture
+// @Description  Upload profile picture
+// @Tags         users
+// @Param Token header string true "Session token is required"
+// @Param ProfilePicture formData file true "The profile picture file is required"
+// @Accept       json
+// @Produce      json
+// @Success      200 {object} model.JSONSuccessResult{status=string,data=nil} "Successfully uploaded the profile picture"
```

Comment:
There's a mismatch between the Swagger API documentation and the actual response.